guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Web-UI-with-a-Firefox-browser.adoc

```diff
@@ -0,0 +1,32 @@
+[id="Using_IdM_credentials_to_log_in_to_the_{Project}_Web_UI-with-a-Firefox-browser_{context}"]
+= Using IdM credentials to log in to the {ProjectWebUI} with a Firefox browser
+
+This section describes how to use the Firefox browser to log in to your {ProjectWebUI} with your {FreeIPA} (IdM) login and password.
+
+.Prerequisites
+* Your system administrator has enrolled your {ProjectServer} into IdM and configured the server to use IdM for authentication.
+For more information, see {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}#Using_FreeIPA_admin[Using {FreeIPA}] in _{InstallingServerDocTitle}_.
+* An IdM server is running and reachable.
+* The host on which you are using a Firefox browser to log in to the {ProjectWebUI} is an IdM client.
+* You have a valid IdM login and password.
+* {Team} recommends using the latest stable Firefox browser.
+* Your Firefox browser is configured for Single Sign-On (SSO).
+
+.Procedure
+. Open a terminal window.
```

Comment:
Same as above.

Reply:
We try to match file name, anchor, and title as closely as possible; I am unsure how to handle `{ProjectWebUI}` in a file name, so either we go with "WebUI" which is the upstream terminology, or you keep it as is. Both is valid.